RubberduckCodeAnalysis/RubberduckCodeAnalysis.csproj

```diff
@@ -26,10 +26,11 @@
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
     <DebugType>pdbonly</DebugType>
     <Optimize>true</Optimize>
-    <OutputPath>bin\Release\</OutputPath>
+    <OutputPath>bin\Debug\</OutputPath>
```

Comment:
Please revert this change. That is why the build is failing. The DLL must be compiled into Release, regardless of whether we build a debug or a release.